### PR TITLE
feat: Layout for the Accelerometer instrument 

### LINKED ISSUE #992: Implement layout for the Accelerometer instrument
**Actual Behaviour**

No Accelerometer instrument found. It's only in the Sensor instrument named as MPU6050 and only has a graph plot. Parent issue is #991 

**Expected Behaviour**

A new layout to be designed inspired by the existing devices in the market

**Steps to reproduce it**

> None 

**LogCat for the issue**

> None

**Screenshots of the issue**

> None

**Would you like to work on the issue?**

Yes

### LINKED ISSUE #1047: New Accelerometer tile
**Actual Behaviour**

Tile does not has a proper image. It uses the Oscilloscope tile.

**Expected Behaviour**

A new tile which stretches out properly is needed.

**Steps to reproduce it**

In instrument list, check tile image for `Accelerometer`

**LogCat for the issue**

> None

**Screenshots of the issue**

screenshot_2018-06-15-08-25-30

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Vikum94**
Fixes #992
Fixes #1047 

Changes:
New layout using a recycler view to view acceleration in three axes 

Screenshots for the change: 
screenshot_2018-06-19-08-00-17

APK for testing: 
[accelerometer_layout_updated.zip](https://github.com/fossasia/pslab-android/files/2109400/accelerometer_layout_updated.zip)

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -56,13 +54,14 @@
...
-
-        <activity android:name=".activity.WaveGeneratorActivity"
-            android:screenOrientation="userLandscape"/>
+        <activity
+            android:name=".activity.MultimeterActivity"
+            android:screenOrientation="portrait" />
+        <activity android:name=".activity.PowerSourceActivity" />
+        <activity
+            android:name=".activity.WaveGeneratorActivity"
+            android:screenOrientation="userLandscape" />
```
Comment (CloudyPadmal) [inline app/src/main/AndroidManifest.xml]: Let's not make changes to code related to other dev's instruments

**Comment by Vikum94** [comment]
@CloudyPadmal Check the new layout

**Comment by Vikum94** [comment]
@Avjeet @harsh-2711 @abhinavraj23 Please review. Thanks!

File: app/src/main/java/org/fossasia/pslab/adapters/AccelerometerAdapter.java [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+        private ImageView axisImage;
+        private TextView value;
+        private TextView maxValue;
+        private TextView minValue;
+        private LineChart chart;
+
+        public ViewHolder(View itemView) {
+            super(itemView);
+            axisImage = itemView.findViewById(R.id.axis_image);
+            value = itemView.findViewById(R.id.accel_value);
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/adapters/AccelerometerAdapter.java]: Can we use `Butterknife` for these bindings?

File: app/src/main/res/layout/accelerometer_list_item.xml [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+                android:layout_alignParentTop="true"
+                android:layout_centerHorizontal="true"
+                android:layout_gravity="center_vertical|center_horizontal|center"
+                android:foregroundGravity="center_vertical"
+                android:gravity="center_vertical|center_horizontal|center"
+                android:text="@string/time"
+                android:textColor="#ffff"
+                android:textSize="@dimen/accel_textsize_small"
+                android:textStyle="normal|bold"
+                tools:layout_editor_absoluteX="288dp"
```
Comment (CloudyPadmal) [inline app/src/main/res/layout/accelerometer_list_item.xml]: Can we put these in `dimens.xml` file?

**Comment by Vikum94** [comment]
@harsh-2711 @CloudyPadmal @abhinavraj23 @Avjeet Changes done. Please review. Thanks.

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -120,6 +120,21 @@
...
 
+    <string name="accelerometer">Accelerometer</string>
+    <string name="x_axis">X axis</string>
+    <string name="y_axis">Y axis</string>
+    <string name="z_axis">Z axis</string>
+    <string name="text_max">Max</string>
+    <string name="text_min">Min</string>
+    <string name="test_value">9.80</string>
+    <string name="meters_per_sec">ms<sup><small>-2</small></sup></string>
+    <string name="time_seconds">time (s)</string>
```
Comment (CloudyPadmal) [inline app/src/main/res/values/strings.xml]: Make t capital

File: app/src/main/res/layout/accelerometer_list_item.xml [hunk 5]
```diff
@@ -0,0 +1,217 @@
...
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/first_chart"
+    android:layout_width="match_parent"
+    android:layout_height="175dp"
+    android:layout_below="@+id/top_app_bar_layout"
+    android:layout_marginEnd="@dimen/card_margin"
+    android:layout_marginLeft="@dimen/card_margin"
+    android:layout_marginRight="@dimen/card_margin"
+    android:layout_marginStart="8dp"
```
Comment (Avjeet) [inline app/src/main/res/layout/accelerometer_list_item.xml]: here also

File: app/src/main/res/layout/accelerometer_list_item.xml [hunk 6]
```diff
@@ -0,0 +1,217 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.v7.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/first_chart"
+    android:layout_width="match_parent"
+    android:layout_height="175dp"
```
Comment (Avjeet) [inline app/src/main/res/layout/accelerometer_list_item.xml]: replace them with dimen values.

**Comment by CloudyPadmal** [comment]
@Vikum94 can you update the screenshot with latest update?

**Comment by Vikum94** [comment]
@CloudyPadmal Conflicts fixed and image updated.

**Comment by Vikum94** [comment]
@Avjeet @harsh-2711 Please help in reviewing this. I cannot move on with work as this is still here. Thank you

**Pull request merged**